app-backend/db/src/test/scala/com/azavea/rf/database/TaskDaoSpec.scala

```diff
@@ -328,32 +330,53 @@ class TaskDaoSpec
               // have to delete actions on the task to be able to delete it
               _ <- fr"TRUNCATE TABLE task_actions;".update.run
               delete <- TaskDao.deleteTask(collection.features.head.id)
+              annoProjAfterDelete <- AnnotationProjectDao
+                .unsafeGetById(dbAnnotationProj.id)
```

Comment:
Can you tack on a delete here for all of the project's tasks? As is this is only testing the case that was working correctly before. Something like

```
_ <- TaskDao.query.filter(fr"annotation_project_id = ${dbAnnotationProj.id}")
annoProjAfterBigDelete <- AnnotationProjectDao.unsafeGetById(dbAnnotationProj.id)
```

and then assert on the final result.